cmd/kubeadm/app/cmd/token.go

```diff
@@ -247,7 +247,7 @@ func RunCreateToken(out io.Writer, client clientset.Interface, cfgPath string, i
 	// In fact, we don't do any CRI ops at all.
 	// This is just to force skipping the CRI detection.
 	// Ref: https://github.com/kubernetes/kubeadm/issues/1559
-	initCfg.NodeRegistration.CRISocket = kubeadmconstants.DefaultDockerCRISocket
+	initCfg.NodeRegistration.CRISocket = "/var/run/unknown.sock"
```

Comment:
Hey @pacoxu, If we don't want to use the value from DefaultDockerCRISocket constant. Shouldn't we create a new constant (generic) so more code can be re-used?

I.e: 
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/cmd/phases/init/waitcontrolplane.go#L110

Reply:
add new constant for this.